avail-core/build.gradle.kts

```diff
@@ -103,6 +105,10 @@ tasks {
 		}
```

Comment:
Shouldn't this also be updated to "/"?  This would put All_Primitives.txt into the indicated subdirectory within the jar, but it looks like you're now expecting it at "/" in the jar.